src/services/TitleService.js

```diff
@@ -0,0 +1,16 @@
+import Service from '../abc/Service';
+export default class TitleService extends Service {
+	constructor(app, serviceName="TitleService"){
+		super(app, serviceName);
+		this.App = app;
```

Comment:
@Aringoaway Ok, I thought you will declare the `this.title` variable in constructor, which you should. I think it should be an empty string. `this.title = ""`